src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/ClaimController.java

```diff
@@ -100,7 +102,18 @@ public Claim save(
         @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
         @RequestHeader(value = "Features", required = false) List<String> features
     ) {
-        return claimService.saveClaim(submitterId, claimData, authorisation, features);
+        return claimService.saveClaim(submitterId, claimData, authorisation, features, CREATE_CASE);
+    }
+
+    @PostMapping(value = "/{submitterId}/create-legal-rep-claim", consumes = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    @ApiOperation("Creates a new legal rep claim")
+    public Claim saveLegalRepClaim(
+        @Valid @NotNull @RequestBody ClaimData claimData,
+        @PathVariable("submitterId") String submitterId,
+        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
+        @RequestHeader(value = "Features", required = false) List<String> features
```

Comment:
Do we need features header for legal rep claims?

Reply:
this is something to confirm but i removed it for now in create legal rep claim service

Reply:
discussed with @ckalista , we agreed to not have features for legal rep claims for now, we will add later if and when we need.